pkg/osquery/table/onepassword_config.go

```diff
@@ -43,45 +71,60 @@ func (o *onePasswordAccountConfig) generateForPath(ctx context.Context, path str
 
 	db.Exec("PRAGMA journal_mode=WAL;")
 
-	rows, err := db.Query("SELECT user_email FROM accounts")
+	rows, err := db.Query("SELECT user_email, team_name, user_first_name, user_last_name, account_type FROM accounts")
```

Comment:
I'm skeptical `team_name` is the correct column. To mirror slack, maybe `server` ? Or `account_uuid`